src/utils/runWorkers.ts

```diff
@@ -0,0 +1,46 @@
+const tokenWorker = new Worker(new URL('../worker/refreshWorker.ts', import.meta.url), {type: 'module'});
+
+const stopAutoRefreshToken = () => {
+    tokenWorker.postMessage({
+        'message': 'stop'
+    });
+}
+
+const startAutoRefreshToken = () => {
+    // wait 30 second before starting the worker
```

Comment:
```suggestion
    // wait before starting the worker
```
Selon ce que je lis dans le code (10s)